src/app/screens/reset-password-screen/reset-password-screen.routing.module.ts

```diff
@@ -0,0 +1,22 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import { ResetPasswordViewComponent } from './reset-password-screen.component';
+
+const routes: Routes = [
+    {
+        path: ':token',
+        component: ResetPasswordViewComponent,
+        pathMatch: 'full'
+    },
```

Comment:
Faz sentido ser uma rota de Guest?
`canActivate: [GuestGuardService]`

Reply:
Penso que sim. O que decide isso? se a route for "fora do login"? Simplesmente fiz assim porque é o que está em `user-activation-screen-routing.module.ts` também. Se se mudar faz sentido mudar noutro, certo?